packages/composables/src/useBilling/index.ts

```diff
@@ -1,14 +1,28 @@
 import { useBillingFactory, UseBillingParams, Context } from '@vue-storefront/core';
 import { Address } from '@vue-storefront/spree-api';
+import useCart from '../useCart';
 
 const params: UseBillingParams<Address, any> = {
+  provide() {
+    return useCart();
+  },
+
   load: async (context: Context): Promise<Address> => {
-    const checkout = await context.$spree.api.getCheckout();
-    return checkout.billingAddress;
+    return context.cart.value.address.billing;
```

Comment:
Can this fail at any point? Does the "dummy" cart contain these fields?

Reply:
Dummy cart and deserialised cart have these fields so it shouldn't fail at any point